ie_serving/main.py

```diff
@@ -145,16 +144,20 @@ def parse_one_model(args):
         logger.info("Could not access provided model. Server will exit now.")
         sys.exit()
 
-    total_workers_number = args.grpc_workers
     if args.rest_port > 0:
-        total_workers_number += args.rest_workers
         process_thread = threading.Thread(target=start_web_rest_server,
                                           args=[models, args.rest_port,
                                                 args.rest_workers])
         process_thread.setDaemon(True)
         process_thread.start()
-    start_server(models=models, max_workers=args.grpc_workers,
-                 port=args.port)
+
+    num_servers = int(os.getenv("NUM_SERVERS", 1))
+    server_processes_handlers = []
+    for i in range(num_servers):
+        server_process_handler = multiprocessing.Process(
+            target=start_server, args=(models, args.grpc_workers, args.port))

```

Comment:
Server processes will not be aware of the models. So variable models among start_server arguments is unnecessary.